validator/src/main/java/org/ethereum/beacon/validator/BeaconChainValidator.java

```diff
@@ -46,11 +44,11 @@
   private MessageSigner<BLSSignature> messageSigner;
 
   /** Validator index. Assigned in {@link #init(BeaconState)} method. */
-  private ValidatorIndex validatorIndex = ValidatorIndex.MAX;
+  @VisibleForTesting ValidatorIndex validatorIndex = ValidatorIndex.MAX;
```

Comment:
I don't like this thing
We have java's private/package-only/protected/public and this thing breaks the things forcing it package-only instead of private
@Nashatyrev  what do you think?

Reply:
Yep, this is something I don't like either. But sometimes for testing this is the only way to run checks easily, without additional tweaks.

Reply:
Yep, looks pretty crappy. 
It's better than declare package-private and put tests to the same package.